When building with support for Large files, the declaration of lseek()
conflicts with that provided by glibc.
--- shadow-20000902/libmisc/utmp.c	Wed Mar 27 20:23:38 2002
+++ shadow-20000902/libmisc/utmp.c	Wed Mar 27 20:23:27 2002
@@ -54,7 +54,7 @@
 extern	void	endutent();
 extern	time_t	time();
 extern	char	*ttyname();
-extern	long	lseek();
+/* extern	long	lseek(); */
 
 #define	NO_UTENT \
 	"No utmp entry.  You must exec \"login\" from the lowest level \"sh\""
